### DISCUSSION TITLE
'd' in Element tree

### DISCUSSION THREAD

**Discussion by Childhome** [pimcore/pimcore#13294] [Q&A / Support] [answered]
I've updated to Pimcore X yesterday.
I saw this 'd' in the Element tree. It doesn't show up on my staging server, only on production.
Maybe a silly question, but what is it and what's the use ?
![image](https://user-images.githubusercontent.com/58265975/193566613-b19e55f0-723e-4895-8305-1855d26c3ecf.png)

**Comment by kingjia90** [comment]
Can you please right-click and `Inspect` this `d` element in the developer tools (F12) and share the expanded HTML of it?
It doesn't look like coming from Pimcore (eg. the font).

**Comment by alexej-d** [comment, marked as answer]
For me this happens with old CSS in the browser cache. Clearing the browser cache fixes this.
